automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -74,7 +74,6 @@ final class LeonardoSuite
       new NotebookDataSyncingSpec,
       new NotebookGATKSpec,
 //      new NotebookHailSpec, TODO: revisit and enable this once hail image https://broadworkbench.atlassian.net/browse/IA-1237
-      new NotebookLocalizeFileSpec,
```

Comment:
Let's disable this test. My rationale:
1. It's failing frequently in Jenkins, lowering our pass percentage and affecting our velocity
2. It's legacy code, meaning we are not maintaining it even if it is flaky
3. It may be running on some old clusters, but we are not deploying it to any _new_ clusters
4. It's used in FireCloud, which is also legacy and IMO doesn't need to be tested